pytradfri/api/aiocoap_api.py

```diff
@@ -41,7 +38,7 @@ def _patched_datagram_received(self, data, addr):
 
 
 @asyncio.coroutine
-def api_factory(host, security_code):
+def api_factory(host, security_code, loop=asyncio.get_event_loop()):
```

Comment:
You should default it to `None` because the call to `get_event_loop` will be cached